graphein/ml/datasets/torch_geometric_dataset.py

```diff
@@ -41,12 +41,12 @@ def __init__(
         self,
         root: str,
         name: str,
-        pdb_paths: Optional[List[str]] = None,
-        pdb_codes: Optional[List[str]] = None,
-        uniprot_ids: Optional[List[str]] = None,
-        graph_label_map: Optional[Dict[str, torch.Tensor]] = None,
-        node_label_map: Optional[Dict[str, torch.Tensor]] = None,
-        chain_selection_map: Optional[Dict[str, List[str]]] = None,
+        pdb_paths: Optional[List[str]] = [],
```

Comment:
What's the reasoning for using empty lists as the default arg?

Reply:
`empty lists` can add together even if they are empty, while `None` can't. So we can skip some `if` for different statements of the user pass `pdb_paths` or `pdb_codes` or `uniprot_ids`, and just merge them into `self.structures,` which is used at `process` func and it works like `os.listdir(self.raw_dir)`.

 As for some potential bugs, i'm really not sure would this will cause some bugs as i use `empty list` instead of `None`.

Reply:
I think this should be `None`

https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists

If you want to retain the behaviour inside the object, you could do:

```python
if working_list is None: working_list = []
```

Reply:
> 

Yes, and actually i have done that in the latest commit